gobblin-cluster/src/main/java/gobblin/cluster/GobblinClusterUtils.java

```diff
@@ -57,4 +64,52 @@ public static Path getAppWorkDirPath(FileSystem fs, String applicationName, Stri
   public static String getAppWorkDirPath(String applicationName, String applicationId) {
     return applicationName + Path.SEPARATOR + applicationId;
   }
+
+  /**
+   * state stores used for storing work units and task states
+   */
+  public static class StateStores {
```

Comment:
I don't see anything cluster specific in this class. Eventually, we may want to use something like that in non-cluster use-cases. Further, I think it is important enough to be a top-level class.